src/findEntryPoints.js

```diff
@@ -3,9 +3,10 @@ var utils = require('./utils');
 
 function isValidFile (file, filePath, packageName) {
   return (
-    (path.extname(file) === '.js' || path.extname(file) === '.css') &&
+    (path.extname(file) === '.js' || path.extname(file) === '.css' || path.basename(file) === 'package.json') &&
     file[0] !== '_' &&
     file !== 'gulpfile.js' &&
+    !file.match(/webpack/) &&
```

Comment:
I saw that some packages has their webpack.config.js file in there, which might require Node packages that just breaks bundle or increases bundle size alot :-)